test/suites/edgexfoundry/proxy_test.go

```diff
@@ -24,59 +28,71 @@ func TestProxy(t *testing.T) {
 		utils.Exec(t, `rm -rf `+CA_KEY_FILE+` `+CA_CERT_FILE+` `+SERVER_KEY_FILE+` `+SERVER_CSR_FILE+` `+SERVER_CERT_FILE+` `+EDGEXFOUNDRY_SNAP_DATA)
 	})
 
-	// Due to confinement issues when running this test, we write the private key to SNAP_DATA
-	// Create private and public keys
+	// Due to confinement issues when running this test, we write the files to SNAP_DATA
 	utils.Exec(t, `sudo mkdir -p `+EDGEXFOUNDRY_SNAP_DATA)
-	utils.Exec(t, `sudo openssl ecparam -genkey -name prime256v1 -noout -out `+PRIVATE_KEY)
-	utils.Exec(t, `sudo openssl ec -in `+PRIVATE_KEY+` -pubout -out `+PUBLIC_KEY)
 
 	// Read the API Gateway token
 	KONG_ADMIN_JWT, _ := utils.Exec(t, "sudo cat /var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt")
 	require.NotEmpty(t, KONG_ADMIN_JWT)
 
-	t.Run("add proxy admin user", func(t *testing.T) {
-		// Use secrets-config to add a user example with id 1000
-		// On success, the above command prints the user id
-		stdout, _ := utils.Exec(t, `
+	// Create private and public keys
+	utils.Exec(t, `sudo openssl ecparam -genkey -name prime256v1 -noout -out `+PRIVATE_KEY)
+	utils.Exec(t, `sudo openssl ec -in `+PRIVATE_KEY+` -pubout -out `+PUBLIC_KEY)
+
+	// Use secrets-config to add a user example with id 1000
+	// On success, the above command prints the user id
+	stdout, _ := utils.Exec(t, `
 			sudo edgexfoundry.secrets-config proxy adduser \
 			--token-type jwt --user example --algorithm ES256 \
 			--public_key `+PUBLIC_KEY+` --id 1000 -jwt `+KONG_ADMIN_JWT)
-		require.Equal(t, "1000\n", stdout)
+	require.Equal(t, "1000\n", stdout)
+
+}
+
+func TestTLSCert(t *testing.T) {
+
+	// start clean
+	utils.Exec(t, `rm -rf `+CA_KEY_FILE+` `+CA_CERT_FILE+` `+SERVER_KEY_FILE+` `+SERVER_CSR_FILE+` `+SERVER_CERT_FILE+` `+EDGEXFOUNDRY_SNAP_DATA)
+
+	t.Cleanup(func() {
+		utils.Exec(t, `rm -rf `+CA_KEY_FILE+` `+CA_CERT_FILE+` `+SERVER_KEY_FILE+` `+SERVER_CSR_FILE+` `+SERVER_CERT_FILE+` `+EDGEXFOUNDRY_SNAP_DATA)
 	})
 
-	t.Run("set custom TLS cert", func(t *testing.T) {
-		certGenerator(CA_KEY_FILE, CA_CERT_FILE, SERVER_KEY_FILE, SERVER_CSR_FILE, SERVER_CERT_FILE)
+	// Due to confinement issues when running this test, we write the files to SNAP_DATA
+	utils.Exec(t, `sudo mkdir -p `+EDGEXFOUNDRY_SNAP_DATA)
 
-		// Setting security-proxy certificate
-		utils.SnapSet(t, "edgexfoundry", "env.security-proxy.tls-certificate", SERVER_CERT_FILE)
+	// Read the API Gateway token
+	KONG_ADMIN_JWT, _ := utils.Exec(t, "sudo cat /var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt")
+	require.NotEmpty(t, KONG_ADMIN_JWT)
 
-		// Setting security-proxy certificate private key
-		utils.SnapSet(t, "edgexfoundry", "env.security-proxy.tls-private-key", SERVER_KEY_FILE)
+	certGenerator(CA_KEY_FILE, CA_CERT_FILE, SERVER_KEY_FILE, SERVER_CSR_FILE, SERVER_CERT_FILE)
+	SERVER_CERT, _ := utils.Exec(t, `sudo cat `+SERVER_CERT_FILE)
+	SERVER_KEY, _ := utils.Exec(t, `sudo cat `+SERVER_KEY_FILE)
 
```

Comment:
There are mixed conventions for variable and constant names. Try to keep these consistent across the whole package.

Local variables, in particular, should use camleCase.

Reply:
You may also wanna define the constants at a needed scope. If they are used by one function only, define them there.

Otherwise there is a chance of naming collision in the package.